.github/workflows/pypi-release.yml

```diff
@@ -20,6 +20,14 @@ jobs:
 
       - name: install deps
         run: pip install --upgrade build twine
+      
+      - name: install frontend deps
+        run: yarn install --frozen-lockfile
+        working-directory: invokeai/frontend
+      
+      - name: build frontend
+        run: yarn build
+        working-directory: invokeai/frontend
```

Comment:
I already thought about something like this, but for now this woudl mean that the frontend could differ between the PyPI Package and the GH-Release, which imho is not a good idea

Reply:
if building on the same codebase it should be a deterministic build that has the same functionality

Reply:
Y'all I forgot to mention the problem is that the main branch is protected and actions cannot push to it. 

We have an discussion ongoing internally on discord about a solution

Reply:
Yeah, my main opinion here is that the frontend is as much of a distribution build as the wheels and sdists, and they probably shouldn't be tracked in the repository.

Reply:
Good point. We didn't have an actual distribution of the application until very very recently.